core-concepts.md

```diff
@@ -6,47 +6,44 @@ number of client nodes, which are used for running jobs. You must also have a [C
 deployed (see the [Consul AWS Module](https://github.com/hashicorp/terraform-aws-consul)) in one of the following
 configurations:
 
-1. [Deploy Nomad and Consul in the same cluster](#deploy-nomad-and-consul-in-the-same-cluster)
-1. [Deploy Nomad and Consul in separate clusters](#deploy-nomad-and-consul-in-separate-clusters)
+  1. [Deploy Nomad and Consul in the same cluster](#deploy-nomad-and-consul-in-the-same-cluster)
+  2. [Deploy Nomad and Consul in separate clusters](#deploy-nomad-and-consul-in-separate-clusters)
```

Comment:
The repeated use of `1.` is intentional; Markdown automatically converts it to the proper number sequence.

Reply:
My VS code reformats lists and messes it up. I'll fix it